### PR TITLE
UI updates for workers beta

### PR DESCRIPTION
**Pull request by desertaxle**
- Adds code banner to work pool page
- Updates work pool queue code banner to show worker start command when appropriate
- Adds workers table behind enable-workers feature flag

### Checklist

- [ ] This pull request references any related issue by including "closes `<link to issue>`"
	- If no issue exists and your change is not a small fix, please [create an issue](https://github.com/PrefectHQ/prefect/issues/new/choose) first.
- [ ] This pull request includes tests or only affects documentation.
- [x] This pull request includes a label categorizing the change e.g. `fix`, `feature`, `enhancement`

### CODE REVIEW

**Review by pleek91 — APPROVED** [review]
Two suggestions. Neither blocking. But I would abstract the code banner stuff in a future PR.

File: ui/src/pages/WorkPool.vue [hunk 1]
```diff
@@ -49,6 +63,16 @@
...
+  const isAgentWorkPool = computed(() => workPool.value?.type === 'prefect-agent')
+
+  const codeBannerTitle = computed(() => {
+    if (!workPool.value) {
+      return 'Your work pool is ready to go!'
+    }
+    return `Your work pool ${workPool.value.name} is ready to go!`
+  })
+  const codeBannerCliCommand = computed(() => `prefect ${isAgentWorkPool.value ? 'agent' : 'worker'} start --pool ${workPool.value?.name}`)
+  const codeBannerSubtitle = computed(() => `Work pools organize work that ${isAgentWorkPool.value ? 'Prefect agents' : 'Prefect workers'} can pull from.`)
```
[the remark this answers is not in the document]
Comment (pleek91) [reply ui/src/pages/WorkPool.vue:75]: Realizing the language is slightly different in WorkPoolQueue.vue. So I might make two banner components. One for WorkPool and one for WorkPoolQueue. That way nebula-ui and oss ui can share them.

File: ui/src/pages/WorkPool.vue [hunk 2]
```diff
@@ -1,42 +1,56 @@
...
   import { usePageTitle } from '@/compositions/usePageTitle'
 
   const api = useWorkspaceApi()
   const workPoolName = useRouteParam('workPoolName')
+  const can = useCan()
 
   const tabs = computed(() => {
     const values = ['Runs', 'Work Queues']
 
+    if (can.access.workers && !isAgentWorkPool.value) {
```
[the remark this answers is not in the document]
Comment (desertaxle) [reply ui/src/pages/WorkPool.vue]: Nice! I updated this component to use `useTabs`.

File: ui/src/pages/WorkPool.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (desertaxle) [reply ui/src/pages/WorkPool.vue]: Good suggestions! I'll make a note to implement those changes in prefect-ui-library.

**Pull request merged**